garage/misc/wrappers.py

```diff
@@ -0,0 +1,88 @@
+"""Gym env wrappers"""
+import cv2
```

Comment:
OpenCV is an enormous dependency. Is there no other Python library which can do this (e.g. PIL or just plain numpy)?

Reply:
How about scipy?